### DISCUSSION TITLE
Python and TagLab issue on macOS

### DISCUSSION THREAD

**Discussion by jjherlan** [cnr-isti-vclab/TagLab#134] [General]
A lab tech and I are trying to install TagLab on macOS. We cannot run TagLab on macOS because the version of Python is currently 3.12.X. What is the best way to install and use Python 3.9.X without uninstalling Python 3.12.X? Thank you in advance.

**Comment by Jordan-Pierce** [comment]
Use an Anaconda environment, with the python version within the conda environment set to 3.8

https://docs.anaconda.com/free/miniconda/
